### ISSUE TITLE
Instructions for building a redistributable Update Site

### ISSUE THREAD

**Issue by abelgomez** [lhillah/pnmlframework#3]
I searched through the repository, and I couldn't find any instructions to build PNML Framework as a redistributable Update Site.

Could you give me some insights on how to do it or add a small how-to to the Update Site module?

**Comment by lhillah** [comment]
Hi Abel, 

I will put a readme in the pnmlFw-Releng project. The main maven config files to build the project are in place there.
Version 2.2.9 is now released in Maven Central. Next, I will put the update site artefacts on our server.

Cheers
Lom

**Comment by lhillah** [comment]
No longer running the tests, too many, no more relevant, and no new ones but the test projects are still there.

**Comment by abelgomez** [comment]
Nice, now I was able to compile everything. I don't know what I was doing wrong, because I checked the build.sh file in the root (that has similar instructions) and anyway I was having problems. Now everything seems solved.

Nevertheless, I found some small problems with the version number of `fr.lip6.pnml.framework.sdk`. I manually updated `pnmlFwFeature-All-In-One/feature.xml`, `PnmlFramework_UpdateSite/site.xml`, `PnmlFramework_UpdateSite/category.xml` and everything run fine.
